__tests__/errors.test.js

```diff
@@ -105,3 +105,66 @@ describe("POST /api/articles/:article_id/comments", () => {
         })
     })
 })
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("400 responds when valid path but invalid id", () => {
+        return request(app)
+        .get("/api/articles/invalidId")
+        .expect(400)
+        .then(( { body }) => {
+            expect(body.msg).toBe("invalid input");
+        })
+    })
+    test("404 responds when valid id but is non-existent", () => {
+        return request(app)
+            .get("/api/articles/333333")
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("article 333333 does not exist");
+            });
+    });
```

Comment:
Same here

Reply:
This should be handled using a custom error. check the length of the RETURNING * rows. If its < 0, throw the error :)